### DISCUSSION TITLE
Convert SVG as character vector into image in table

### DISCUSSION THREAD

**Discussion by Aariq** [rstudio/gt#1523] [Q&A] [answered]
I have a dataframe with a column `image` that contains `<svg>` tags.  How can I get these to render as the actual image encoded by the SVG in a `gt` table instead of just the text?

**Comment by rich-iannone** [comment, marked as answer]
This'll sound strange (and there really should be a `fmt_html()` function) but could you try using `fmt_markdown()` on that column?

**Reply by Aariq** [reply]
That does the trick! Thanks!
